contrib/examples/apiserver/apiserver.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: service-catalog-apiserver
+  labels:
+    app: apiserver
+spec:
+  containers:
+  - name: apiserver
+    image: apiserver
+    imagePullPolicy: IfNotPresent
+    args: 
+    - --etcd-servers
+    - http://localhost:2379
+    - -v
+    - "10"
+    ports:
+    - name: https
+      containerPort: 6443
+  - name: etcd
+    image: gcr.io/google_containers/etcd:3.0.14
+    command: ["etcd"]
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: service-catalog-apiserver
+spec:
+  selector:
+    app: apiserver
+  ports:
+  - protocol: TCP
+    port: 6443
```

Comment:
Two reasons:
- the [api-server-walkthrough](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/api-server-walkthrough.md) uses 6443 
- the k8s api server usually uses 8443, so 6443 makes it aligned with that